exercises/practice/palindrome-products/palindrome-products.spec.js

```diff
@@ -79,7 +79,7 @@ describe('Palindromes', () => {
     expect(sortFactors(smallest.factors)).toEqual(expected.factors);
   });
 
-  test.skip('largest palindrome from four digit factors', () => {
+  xtest('largest palindrome from four digit factors', () => {
```

Comment:
Pretty sure this was intentional.

Reply:
I think the previous solution caused some error when ran into this test case, so the maintainer skipped this test by `test.skip`. After the changes, it should work out fine, so I reverted `test.skip` to `xtest`.

Reply:
This is on purpose the test runner on the website is not supposed to run this because it can take too long.  There is a bug now where instead it fails.  That's a separate issue.